lua/autorun/server/sv_chip_lister.lua

```diff
@@ -134,7 +136,10 @@ local function getVisibleListUsers()
     if listUserCount == 0 then return {}, 0 end
 
     local listers = ents.FindByClass( "cfc_chip_lister" )
-    if #listers == 0 then return {}, 0 end
+
+    if #listers == 0 then
+        listers = false
```

Comment:
`getVisibleListUsers` shouldn't be looping over all the players when no listers are found, this should return. Type mixing should also not be done

Reply:
The whole point of this PR is to make it still loop through the players when no lister is found. Players should not have to spawn a lister entity in order to use the HUD.